test/integration/play/test_play.py

```diff
@@ -1,14 +1,60 @@
 import os
+import platform
 import re
 import shutil
+import subprocess
+import tempfile
 
 import pytest
+from integration.connect.test_connect import (start_app, stop_app,
+                                              try_execute_on_instance)
 
-from utils import TarantoolTestInstance, run_command_and_get_output
+from utils import (TarantoolTestInstance, control_socket, create_tt_config,
+                   initial_snap, initial_xlog, lib_path,
+                   run_command_and_get_output, run_path, wait_file)
 
 # The name of instance config file within this integration tests.
 # This file should be in /test/integration/play/test_file/.
 INSTANCE_NAME = "remote_instance_cfg.lua"
+BINARY_PORT_NAME = "tarantool.sock"
+
+
+def get_version(tt_cmd, tmpdir):
+    run_cmd = [tt_cmd, "run", "-v"]
+    instance_process = subprocess.run(
+        run_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    if instance_process.returncode == 0:
+        stdout = instance_process.stdout
+        full = stdout.splitlines()[0]
+        for word in re.split(r'\s', full):
+            matched = re.match(r'^\d+\.\d+\.\d+', word)
+            if matched:
+                print("Matched:")
+                print(matched)
+                version = re.split(r'\.', matched.group(0))
+                return True, int(version[0]), int(version[1]), int(version[2])
+    return False, 0, 0, 0
+
+
+def is_cluster_app_supported(tt_cmd, tmpdir):
+    ok, major, minor, patch = get_version(tt_cmd, tmpdir)
+    assert ok
+    return major >= 3
+
+
+def skip_if_cluster_app_unsupported(tt_cmd, tmpdir):
+    if not is_cluster_app_supported(tt_cmd, tmpdir):
+        pytest.skip("Tarantool 3.0 or above required")
```

Comment:
Do we have helpers for it in utils? We should have it, or it should be added to a some common place.

Reply:
Used directly from `connect.py`.

Reply:
It's better to move it into utils.py than.